internal/report/report.go

```diff
@@ -1089,6 +1130,30 @@ func reportLabels(rpt *Report, g *graph.Graph, origCount, droppedNodes, droppedE
 	return label
 }
 
+func parseTagFocusIgnore(tag string) string {
```

Comment:
I think trying to parse the tag filter expression makes things more complicated than needed. Also, the tag ranges can be used for tags other than allocations related so the "with allocations" part may easily get wrong.